lock-api/src/main/java/com/palantir/lock/LockServerOptions.java

```diff
@@ -140,6 +141,10 @@ protected final Object writeReplace() {
         return new SerializationProxy(this);
     }
 
+    public String getLockStateLoggerDir() {
```

Comment:
this should be moved up to be by other params in this config file.  Not really clear to me why this has a special equals / hashCode etc, probably want to add this param there as well.